### PR TITLE
Made "Beverage 'undead'" artifact trigger hint clearer

### PR DESCRIPTION
**Pull request by TheGrimbeeper**
Saw people insisting in game that it had to be zombie blood. Tweaked hint to make it clearer that its a drink, not something that requires a specific gamemode???

**Changelog**

:cl: 
- tweak: Changed hint for a beverage artifact trigger to be clearer.

### CODE REVIEW

**Comment by hivehum** [comment]
ive noticed in general people seem really confused at what to do for the beverage trigger. maybe reword the full hint to not just be "beverage"? idk what would be more clear though

**Review by formlessnameless — APPROVED** [review]
